data/Nelsonomotuyole.json

```diff
@@ -2,8 +2,7 @@
   "name": "Nelson omotuyole",
   "displapstatsPublic": true,
   "type": "personal",
-  "bio": "Teacher turned **programmer** | Intergrated science student from *The university of lagos Nigeria | presently Learning **python pro bootcamp with angela yu**",
-  "avatar": "https://github.com/Nelsonomotuyole.png",
```

Comment:
Seems like you have removed the avatar field too, it will then look like this. 👇
![image](https://user-images.githubusercontent.com/88786642/212832231-4cc7f435-7883-4285-b914-66c65672231f.png)
<hr/>

I recommend you to add the avatar field back, you can use the code below. 🙂
```
"avatar": "https://github.com/Nelsonomotuyole.png",
```

Reply:
Thanks bro, done